src/controllers/propertyInfo/propertyController.js

```diff
@@ -0,0 +1,83 @@
+import Model from '../../models/model';
+
+const propertyModel = new Model('properties');
+
+// eslint-disable-next-line consistent-return
+export const createProperty = async (req, res) => {
+  const { id } = req.user.newUser;
+  const {
+    image, title, address, landArea, noOfRoom, noOfBath, noOfGarage, noOfStore, yearBuild, purpose
+  } = req.body;
+  const columns = 'agent_id, image_url, title, address, land_area, no_of_rooms, no_of_bathrooms, no_of_garage, no_of_store, year_of_build, purpose';
+  const values = `'${id}', '${image}', '${title}', '${address}', '${landArea}', '${noOfRoom}', '${noOfBath}', '${noOfGarage}', '${noOfStore}', '${yearBuild}', '${purpose}' `;
+  try {
+    const data = await propertyModel.insertWithReturn(columns, values);
+    res.status(201).json(data.rows);
+  } catch (err) {
+    return res.status(500).json({ messages: err.stack.messages });
+  }
+};
+
+// eslint-disable-next-line consistent-return
+export const allProperties = async (req, res) => {
+  try {
+    const getProperties = await propertyModel.select('*');
+    if (getProperties.rows === 0) {
```

Comment:
How can `properties.row` equates 0, I thought `row` is an array, do you mean `properties.rows.length` ?